### PR TITLE
Added shimmer animation

### LINKED ISSUE #17: Add shimmer animation to recycler views
**Is your feature request related to a problem? Please describe.**
Using a **progress bar** to show loading status in recycler views does not reflect good user experience. 

**Describe the solution you'd like**
Add shimmer animation similar to how Facebook does, using their library.
Refer to this tutorial for implementing this 
https://medium.com/mindorks/android-design-shimmer-effect-fa7f74c68a93

### PR DESCRIPTION
**Pull request by rob729**
Fixes #17 
Added Shimmer animation in mess, complaint status, resources activity.

### CODE REVIEW

**Comment by rob729** [comment]
@sonusourav Review this PR.

**Comment by sonusourav** [comment]
Make a common template for all shimmer layout something similar to below one and use this in every recycler view.
Shimmer-Feed-Demo
# Why?
Because something like this happens --> Course list is empty.
WhatsApp Image 2019-06-30 at 10 12 44 AM

And anyway displaying item layout does not look good also.
Also, there are some redundant lines of progress dialog left like their declarations. Find and remove them.

**Comment by sonusourav** [comment]
Reformat code before pushing

**Comment by rob729** [comment]
@sonusourav I have done the requested changes.

**Comment by sonusourav** [comment]
@rob729 Could you please squash your commits and make a single commit.

**Comment by sonusourav** [comment]
Also, some unused imports are present. Reformat your code as shown in the picture.
Right click on any file to reformat. Reformat removes your unused import, formats your spacing and indentation and does many other things.
reformat

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -70,4 +70,5 @@
     <color name="detail_color">#0995d1</color>
     <color name="red">#ec2e2e</color>
     <color name="blue">#35d9f2</color>
+    <color name="background">#dddddd</color>
```
Comment (sonusourav) [inline app/src/main/res/values/colors.xml]: Give a proper name to this color, not just background. The name should be relevant to what is its use. Give something like placeholder_back_color or common name of the color ( if any).

File: app/src/main/res/layout/fragment_feedback.xml [hunk 2]
```diff
@@ -22,6 +50,7 @@
 
    </ListView>
 
+
```
Comment (sonusourav) [inline app/src/main/res/layout/fragment_feedback.xml]: Do not add extra lines. Remove everywhere wherever extra lines are present.

**Review by sonusourav — CHANGES_REQUESTED** [review]

**Comment by rob729** [comment]
@sonusourav I have done the requested changes.

**Pull request merged**